Modules/mathmodule.c

```diff
@@ -1072,19 +1072,22 @@ math_2(PyObject *const *args, Py_ssize_t nargs,
 
 FUNC1(acos, acos, 0,
       "acos($module, x, /)\n--\n\n"
-      "Return the arc cosine (measured in radians) of x.")
+      "Return the arc cosine (measured in radians) of x. "
```

Comment:
I don't think there should be a trailing space after the period (the same goes for the other docstrings being modified). Also, I think the period should be followed by `\n\n` in order to conform to [PEP 257](https://www.python.org/dev/peps/pep-0257/) (like the docstring for `atan2()`).
```suggestion
      "Return the arc cosine (measured in radians) of x.\n\n"
```